test/addons/async-hooks-id/binding.cc

```diff
@@ -17,9 +17,17 @@ void GetTriggerAsyncId(const FunctionCallbackInfo<Value>& args) {
     node::AsyncHooksGetTriggerAsyncId(args.GetIsolate()));
 }
 
+void EmitAsyncInit(const FunctionCallbackInfo<Value>& args) {
+  assert(args[0]->IsObject());
+  node::async_uid uid =
+      node::EmitAsyncInit(args.GetIsolate(), args[0].As<Object>(), "foobar");
```

Comment:
The `EmitAsyncInit` that returns `async_id` requires 4 parameters, correct? Or is this a test for the `async_context` casting?

Also, is `node::async_uid` by purpose?